src/UserInfo.js

```diff
@@ -10,7 +10,11 @@ class UserInfo extends Component {
   render() {
     return (
       <div className="UserInfo">
-        <img class="avatar" src={this.props.user && this.props.user.image[0].contentUrl} />
+        <img class="avatar" src={
+            this.props.user && this.props.user.image
+              ? this.props.user.image[0].contentUrl
+              : "http://espace-icare.net/wp-content/uploads/2017/05/noprofile-big.png"
```

Comment:
Hi @souhailmerroun! Thanks for your PR! 👏 🎉 

Could you make a slight change to include this placeholder image in the repo insteads?

Reply:
Done.